Sources/Orbit/Components/Button.swift

```diff
@@ -30,10 +30,11 @@ public struct Button<LeadingIcon: View, TrailingIcon: View>: View {
                 trailingIcon
             }
         )
+        .textSize(custom: size.textSize.value)
```

Comment:
Can we put this inside OrbitButtonStyle?

Reply:
Updated